docs/deployment.rst

```diff
@@ -0,0 +1,144 @@
+.. include:: global.rst.inc
+.. _deployment:
+
+Deployment
+==========
+
+This chapter will give an example how to setup a borg storage server for multiple
+clients.
+
+Machines
+--------
+
+There are multiple machines used in this chapter and will further be aliased by their
+respective fully qualified domain name (fqdn).
+
+* The backup server: `backup01.srv.local`
+* The clients:
+  * John Doe's desktop: `johndoe.clnt.local`
+  * Webserver 01: `web01.srv.local`
+  * Application server 01: `app01.srv.local`
+
+User and group
+--------------
+
+The storage server needs to have only one UNIX user for all the clients.
+Recommended user and group and additional settings:
+
+* User: `backup`
+* Group: `backup`
+* Shell: e.g `/bin/bash` which is capable to run the `borg serve` command
+
+The clients probably will initiate a backup from the root user to catch all
+users, groups and permissions (e.g when backupping `/home`).
+
+Folders
+-------
+
+The following folder tree layout is suggested on the storage server:
+
+* User home directory, /home/backup
+* Repositories path (storage pool): /home/backup/repos
+* Clients restricted paths: `/home/backup/repos/<client fqdn>`
+  * johndoe.clnt.local: `/home/backup/repos/johndoe.clnt.local`
+  * web01.srv.local: `/home/backup/repos/web01.srv.local`
+  * app01.srv.local: `/home/backup/repos/app01.srv.local`
+
+Restrictions
+------------
+
+We need to restric the clients into their paths. Borg needs to be instructed to
+enforce this: ``borg serve --restrict-path /home/backup/repos/<client fqdn>``
+
+Clients can have one or more allowed ssh keys. Albeit recommended is to only use
+one per client. We add keys for ``johndoe.clnt.local``, ``web01.srv.local`` and
+``app01.srv.local``. Only they run under the user ``backup@backup01.srv.local``. A option
+per on-line of ``authorized_keys`` is added to distinguish between the clients:
```

Comment:
Better. But: "an option"

Also, maybe use less passive voice (in general, maybe read through the whole thing to find/fix other places).
"We use per-ssh-key options in authorized_keys file to distinguish between the clients."